src/server/api/data.ts

```diff
@@ -0,0 +1,35 @@
+import axios from "axios";
+import "../config";
+
+interface Coordinates {
+  latitude: number;
+  longitude: number;
+}
+
+const API_KEY = process.env.TICKET_MASTER_API_KEY;
+
+export const getCoordinates = async (postcode: any): Promise<Coordinates> => {
+  const response = await axios.get(
+    `https://api.postcodes.io/postcodes/${postcode}`
+  );
+
+  const lat: number = response.data.result.latitude;
+  const long: number = response.data.result.longitude;
+
+  return {
+    latitude: lat,
+    longitude: long,
+  };
+};
+
+export const getNearestConcert = async (
+  radius: number,
+  unit: string,
+  geohash: any
+): Promise<{}> => {
+  const response = await axios.get(
```

Comment:
You can use generic here when `TicketmasterResponse` is defined.

Reply:
updated code - https://github.com/builtbybea/bass-and-treble/pull/2/commits/40bd1cbf4f7db1364ec51a707c718da0fddf4631